File: app/src/main/java/br/com/testeandroid/model/DownInfo.java
package br.com.testeandroid.model;

import com.google.gson.annotations.SerializedName;

public class DownInfo {

    @SerializedName("name")
    private String name;
    @SerializedName("data")
    private String data;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }
}

File: app/src/main/java/andcordeiro/com/br/testeandroid/entities/DownInfo.kt
package andcordeiro.com.br.testeandroid.entities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class DownInfo:Serializable {

    @SerializedName("name")
    var name: String? = null
    @SerializedName("data")
    var data: String? = null

    override fun toString(): String {
        return "DownInfo(name=$name, data=$data)"
    }
}